bootstrap/kubeadm/api/v1alpha3/types.go

```diff
@@ -0,0 +1,22 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha3
+
+// KubeadmConfigTemplateResource defines the Template structure
+type KubeadmConfigTemplateResource struct {
```

Comment:
Is there a reason to keep this into a separated file? (vs merging into `kubeadmconfigtemplate_types.go`)

Reply:
Not sure... I can move it to `kubeadmconfigtemplate_types.go` if there are no objections.